dotcom-rendering/src/components/Dropdown.tsx

```diff
@@ -362,25 +367,21 @@ const DropdownLink = ({ link, index }: DropdownLinkProps) => {
 					}
 				}}
 			>
-				{link.title}
-				{link.notifications?.map((notification) => (
-					<NotificationMessage
-						notification={notification}
-						key={notification.id}
-					/>
-				))}
-			</a>
-
-			{hasNotifications && (
-				<div
-					css={css`
-						margin-top: 12px;
-						margin-right: 8px;
-					`}
-				>
-					<NotificationBadge diameter={22} />
+				<div>
+					{link.title}
+					{link.notifications?.map((notification) => (
+						<NotificationMessage
+							notification={notification}
+							key={notification.id}
+						/>
+					))}
 				</div>
-			)}
+				{hasNotifications && (
+					<div style={{ display: 'block' }}>
```

Comment:
I think it would be preferable to set `flex-shrink: 0` on the `NotificationBadge`

Reply:
Oh cool, I didn't know about `flex-shrink` 👍

Reply:
That worked well, thanks for the suggestion.